src/pages/questionPage.js

```diff
@@ -23,13 +23,18 @@ export const initQuestionPage = (userInterface) => {
     .addEventListener('click', nextQuestion)
 };
 // check answers if correct or not 
+let isSelectedOneAnswer= false;
 const getAnswer = (e)=> {
   const currentQuestion = quizData.questions[quizData.currentQuestionIndex];
+if(!isSelectedOneAnswer){
   const answer = e.target.innerText;
   currentQuestion.selected=answer.charAt(0)
   currentQuestion.selected===currentQuestion.correct ? console.log('selected correct answer') : console.log('selected wrong answer')
+  isSelectedOneAnswer=true;
+}
 }
 const nextQuestion = () => {
+  isSelectedOneAnswer=false; // I added this code to reset isSelectedOneAnswer for next question
```

Comment:
Have you tested without this line? I think it might not be needed, but I'm not sure.

Reply:
Yes, If I did not add it next question starts with as selected.